common/appscale/common/service_stats/stats_manager.py

```diff
@@ -284,38 +275,36 @@ def _finalize_request(self, request_no):
                              self._cumulative_counters, request_info)
 
   def _increment_counters(self, counters_config, counters_dict, request_info):
-    for counter_pair in iteritems(counters_config):
+    for counter_name, categorizer, summarizer, nested_config in counters_config:
       # Counters config can contain following types of items:
-      #  - str->Matcher
-      #  - Categorizer->Matcher
-      #  - Categorizer->nested config with the same structure
+      #  - str, callable(summarizer)
+      #  - str, callable(categorizer), callable(summarizer)
+      #  - str, callable(categorizer), nested config(tuple)
```

Comment:
Could you explicitly write `None` where value is missing? e.g.:
```
 - str, None,                  callable(summarizer), None
 - str, callable(categorizer), callable(summarizer), None
 - str, callable(categorizer), None,                 tuple(nested config)
``` 
The same for recent stats.

Reply:
Or without columns alignment..